.github/workflows/ci.yaml

```
@@ -11,6 +11,19 @@ jobs:
     #     runtime: [docker, conda]
     # permissions:
     #   id-token: write
-    uses: nextstrain/.github/.github/workflows/pathogen-repo-ci.yaml@master
+    name: nextstrain build ingest
+    uses: ./.github/workflows/pathogen-repo-ci.yaml
```

FWIW, instead of a local fork, you can point to the centralized workflow on an alternative branch, e.g.

```suggestion
    uses: nextstrain/.github/.github/workflows/pathogen-repo-ci.yaml@update-pathogen-repo-ci-63
```

---

Thanks — I was iterating pretty rapidly on both the underlying "base" workflow and the ci.yaml in this repo so it was faster to just be able to keep amending the commit and force pushing the one repo. (I'm not particularly _happy_ about that but I think if I was force pushing branches in 2 repos at the same time where one was dependent on the other, I would have tripped over my own feet a lot ...more.)